snaps/features/custom-name-resolution.md

```diff
@@ -78,9 +78,10 @@ export const onNameLookup: OnNameLookupHandler = async (request) => {
 ```
 
 :::note
-The response from the `onNameLookup` handler includes a `domainName` property. This can
-be used to do fuzzy matching on domain names, by returning the domain that was resolved rather than
-the one that was passed in.
+
+The `onNameLookup` handler response includes a `domainName` property.
+You can use this property to perform fuzzy matching on domain names by returning the resolved domain rather than the one provided in the request.
```

Comment:
```suggestion
You can use this property to perform fuzzy matching on domain names, by having the handler return the resolved domain rather than the one provided in the request.
```